src/templates/middleware.js

```diff
@@ -11,7 +11,15 @@ middleware.nuxti18n = async (context) => {
 
   // Handle root path redirect
   if (route.path === '/' && rootRedirect) {
-    redirect('/' + rootRedirect, route.query)
+    let statusCode = 302
+    let path = rootRedirect
+
+    if (typeof rootRedirect !== 'string' && rootRedirect.path && rootRedirect.statusCode) {
```

Comment:
What's the purpose of checking that properties are set? If they are not, we're gonna pass `rootRedirect` object as path and given that it's not a string, it's not gonna work anyway.

Reply:
Well I did this because of my implementation in the condition block

Reply:
We can't recover from a situation when `rootRedirect` is not a string and those properties are missing so we are as good (or bad) without those checks.

Reply:
If rootRedirect is an object and does not contains both statusCode and path it will throw an undefined error. I though we might not want to throw but just fallback on the default behavior. But if you think it is unnecessary I'll change it :)

Reply:
It will not throw, it will set both `statusCode` and `path` to `undefined`.
And what's the other option anyway, setting `path` to empty object? That won't work either.